.teamcity/_self/KIBProject.kt

```diff
@@ -0,0 +1,19 @@
+package _self
+
+import _self.buildTypes.BuildChainAMI
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+
+
+object KIBProject: Project({
+   val buildTargets = listOf("centos7", "rhel79", "rhel82", "rhel84")
+   val upstreamProjectId = "MesosphereOnly_ClosedSource_SecureSigning_BuildNokmemRpmRepos"
+   for (target in buildTargets) {
+       buildType(BuildChainAMI(target, upstreamProjectId))
+   }
+
+    val fipsBuildTargets = listOf("centos7-fips", "rhel84-fips")
```

Comment:
should we add fips target for `rhel79` or `rhel82` too?

Reply:
Use this as a guide to what should be added: https://docs.d2iq.com/dkp/konvoy/2.2/supported-operating-systems/

Reply:
👍